crates/sui-rpc-loadgen/src/payload/mod.rs

```diff
@@ -90,16 +127,20 @@ impl Command {
         }
     }
 
-    pub fn new_query_transaction_blocks(
-        from_address: Option<String>,
-        to_address: Option<String>,
-    ) -> Self {
-        let query_transactions = QueryTransactions {
-            from_address: from_address.map(|addr| SuiAddress::from_str(&addr).unwrap()),
-            to_address: to_address.map(|addr| SuiAddress::from_str(&addr).unwrap()),
+    pub fn new_query_transaction_blocks<T>(
+        address_type: AddressQueryType,
+        addresses: Option<Vec<T>>,
+    ) -> Self
+    where
+        T: Into<Address>,
```

Comment:
this is because we get Vec<String> from CLAP, Vec<SuiAddress> when loading from `rpc_command_processor.prepare`. Alternatively can always take in `Vec<String`> (which requires more transformations from rpc_command_processor) or `Vec<SuiAddress>` (which requires `main.rs to do the transformation to SuiAddress). happy to go any direction or a different suggestion altogether